test/e2e/testsuites/dynamically_provisioned_volume_snapshot_tester.go

```diff
@@ -53,9 +62,36 @@ func (t *DynamicallyProvisionedVolumeSnapshotTest) Run(client clientset.Interfac
 	ginkgo.By("checking that the pod's command exits with no error")
 	tpod.WaitForSuccess()
 
-	ginkgo.By("creating volume snapshot class")
+	ginkgo.By("Creating test resource group")
+	creds, err := credentials.CreateAzureCredentialFile(false)
+	framework.ExpectNoError(err, fmt.Sprintf("Error getting creds for AzurePublicCloud %v", err))
+	defer func() {
+		err := credentials.DeleteAzureCredentialFile()
+		framework.ExpectNoError(err)
+	}()
+	azureClient, err := azure.GetAzureClient(creds.Cloud, creds.SubscriptionID, creds.AADClientID, creds.TenantID, creds.AADClientSecret)
+	framework.ExpectNoError(err)
+
+	ctx := context.Background()
+	_, err = azureClient.EnsureResourceGroup(ctx, creds.ResourceGroup, creds.Location, nil)
```

Comment:
so it's still create snapshot on current resource group?

Reply:
It wiil create a new resource group and convert it to volume snapshot class.